### ISSUE TITLE
[Feature]: drop item (almost) anywhere on the window to move it there

### ISSUE THREAD

**Issue by silverhawke249** [Cidan/BetterBags#418]
### Describe the feature

in AdiBags, when moving items between bag/bank or say, character sheet, you can drop the item anywhere on the window (not onto another item) to move it there.

in BetterBags, you have to drop it on the free space slot. i would love to see the same behavior as AdiBags as a toggleable option.

### Describe the implementation

to avoid changing behavior unexpectedly, there should be an option in the settings that toggles this behavior, defaulting to the old behavior.

**Comment by zeptognome** [comment]
I agree with this and would love to see it extended to dropping on tabs directly as well

**Comment by Cidan** [comment]
Duplicate of #408. Tracking the ability to drop items anywhere onto the bag window (rather than only the small free space slot) under #408.
